src/tools/PrizeTierController/SavePrizeTiersModal/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { BottomSheet, Button } from '@pooltogether/react-components'
+import {
+  isSavePrizeTiersModalOpenAtom,
+  savePrizeTiersModalStateAtom,
+  SavePrizeTiersModalState
+  // prizeTierEditsAtom
+} from '@prizeTierController/atoms'
+import { useAtom } from 'jotai'
+
+export const SavePrizeTiersModal = () => {
+  const [isOpen, setIsOpen] = useAtom(isSavePrizeTiersModalOpenAtom)
+  const [modalState, setModalState] = useAtom(savePrizeTiersModalStateAtom)
```

Comment:
Since you're just passing all of the uses of this atom through `onContinue` you should just use `useState` inside this component rather than an atom to keep app memory a little cleaner.

However, if you plan to use this atom somewhere else far away in the app then keep the atom!